packages/messaging/android/src/main/java/com/adobe/marketing/mobile/reactnative/messaging/RCTAEPMessagingModule.java

```
@@ -175,11 +177,30 @@ public void track(final String messageId, final String interaction,
     }
   }
 
+  @ReactMethod
+  public void handleJavascriptMessage(final String messageId, final String handlerName) {
+    Presentable<?> presentable = presentableCache.get(messageId);
+    if (presentable == null || !(presentable.getPresentation() instanceof InAppMessage)) return;
```

If the presentable is not found this just silently returns.
Could you please add logs or could we emit a warning why a handler wasn’t called.